data/preferences/src/components/pages/Preferences.vue

```diff
@@ -146,6 +146,23 @@
           ></editable-text-list>
         </td>
       </tr>
+      <tr>
+        <td class="pull-top">
+          <label for="show-all-apps">Show All Apps</label>
+          <small>
+            <p>
+              Ulauncher will display regular applications no matter the value of OnlyShowIn / NotShowIn
+              from the .desktop files.
```

Comment:
A potential wording change:

> Ulauncher will display all applications, even if they are configured to not be shown in the current desktop environment.

(mostly to drop the reference to the OnlyShowIn/NotShowIn parameters, as someone not familiar with .desktop files might not understand the implications.

Reply:
Fixed